base_tier_validation/models/tier_validation.py

```diff
@@ -469,18 +478,12 @@ def request_validation(self):
                         order="sequence desc",
                     )
                     sequence = 0
+                    vals = []
```

Comment:
For following naming conventions:

```suggestion
                    vals_list = []
```

Reply:
And declare it outside the loop and do only one create.

Reply:
This last comment is unattended.

Reply:
It's changed

Reply:
Outside the other loop, and you can join the 2 `if` lines into one.